equinix/resource_fabric_connection_acc_test.go

```diff
@@ -0,0 +1,998 @@
+
+	return fmt.Sprintf(`resource "equinix_fabric_connection" "test" {
+		type = "EVPL_VC"
+		name = "fabric_tf_acc_test_CCEPL"
+		notifications{
+			type = "ALL"
+			emails = ["test@equinix.com","test1@equinix.com"]
+		}
+		order {
+		purchase_order_number = "1-129105284100"
+			}
+		bandwidth = %d
+		a_side {
+		access_point {
+			type = "COLO"
+				port {
+				uuid = "c4d9350e-77cd-7cdd-1ce0-306a5c00a600"
+				}
+				link_protocol {
+					type= "QINQ"
+					vlan_s_tag= 1231
+				}
+			}
+		}
+		z_side {
+		access_point {
+			type = "COLO"
+				port{
+				uuid = "c4d9350e-783c-83cd-1ce0-306a5c00a600"
+				}
+				link_protocol {
+					type= "QINQ"
+					vlan_s_tag= 457
+				}
+			location {
+        		metro_code= "SV"
+      		}
+			}
+		}
+	}`, bandwidth)
+}
+
+func testAccFabricCreateAzureConnectionConfig(bandwidth int32) string {
+	log.Printf(" inside create connection test config ")
+	return fmt.Sprintf(`resource "equinix_fabric_connection" "test" {
+	name = "fabric_tf_acc_CSAZURE"
+	description = "Test Connection"
+	type = "EVPL_VC"
+	notifications{
+		type="ALL" 
+		emails=["example@equinix.com"]
+	} 
+	bandwidth = %d
+	redundancy {priority= "PRIMARY"}
+	order {
+    	purchase_order_number= "1-323292"
+  	}
+  	a_side {
+    	access_point {
+      		type= "COLO"
+      		port {
+        		uuid= "c4d9350e-783c-83cd-1ce0-306a5c00a600"
+      		}
+      	link_protocol {
+        	type= "QINQ"
+        	vlan_s_tag= "2019"
+      	}
+    }
+  }
+  	z_side {
+    	access_point {
+      		type= "SP"
+			authentication_key= "a38565b9-5d32-45ba-bb01-0649e2735753"
+      		profile {
+        		type= "L2_PROFILE"
+        		uuid= "bfb74121-7e2c-4f74-99b3-69cdafb03b41"
+      		}
+      		location {
+        		metro_code= "SV"
+      		}
+    	}
+  	}
+}
+`, bandwidth)
+}
+
+func testAccFabricUpdateConnectionConfig(bandwidth int32) string {
+	log.Printf(" inside create connection test config ")
+	return fmt.Sprintf(`resource "equinix_fabric_connection" "test" {
+	uuid = equinix_fabric_connection.test.uuid
+	name = "fabric_tf_acc_CSAZURE"
+	description = "Test Connection"
+	type = "EVPL_VC"
+	notifications{
+		type="ALL" 
+		emails=["example@equinix.com"]
+	} 
+	bandwidth = %d
+	redundancy {priority= "PRIMARY"}
+	order {
+    	purchase_order_number= "1-323292"
+  	}
+  	a_side {
+    	access_point {
+      		type= "COLO"
+      		port {
+        		uuid= "c4d9350e-783c-83cd-1ce0-306a5c00a600"
+      		}
+      	link_protocol {
+        	type= "QINQ"
+        	vlan_s_tag= "2019"
+      	}
+    }
+  }
+  	z_side {
+    	access_point {
+      		type= "SP"
+			authentication_key= "a38565b9-5d32-45ba-bb01-0649e2735753"
+      		profile {
+        		type= "L2_PROFILE"
+        		uuid= "bfb74121-7e2c-4f74-99b3-69cdafb03b41"
+      		}
+      		location {
+        		metro_code= "SV"
+      		}
+    	}
+  	}
+}
+`, bandwidth)
+}
+
+/*func testAccFabricCreateConnectionConfig(conectionType string) string {
```

Comment:
do you expect we'll be using these commented-out tests? remove them if not.

Reply:
These were created for testing team, but I took back and removed from here.